cpu/lpc11u34/include/periph_cpu.h

```diff
@@ -62,6 +62,11 @@ typedef uint16_t gpio_t;
 #define GPIO_PIN(port, pin)     (gpio_t)((port << 16) | pin)
 
 /**
+ * @brief   Number of PWM channels per PWM peripheral
+ */
+#define PWM_CHAN_NUMOF      (3U)
```

Comment:
because it is CPU specific, 3 channels is the maximum what the timer provides...